conjurapi/resource_test.go

```diff
@@ -74,7 +25,7 @@ func TestClient_CheckPermission(t *testing.T) {
 		}
 	}
 
-	conjur, err := conjurSetup()
+	conjur, err := conjurSetup(&Config{}, defaultTestPolicy)
```

Comment:
@jtuttle I think it would be good to have a method called `conjurDefaultSetup` which translates to `conjurSetup(&Config{}, defaultTestPolicy)`. Just to avoid the duplication